scripts/create_django_issue.py

```diff
@@ -1,3 +1,12 @@
+# /// script
+# requires-python = ">=3.12"
+# dependencies = [
+#     "gitpython",
+#     "PyGithub",
+#     "jinja2",
+#     "requests",
+# ]
+# ///
```

Comment:
✨ 

Question: does uv try to restrict these versions from the project lock file, or does it just use the versions as specified here?

Reply:
From my understanding it uses the versions specified in the project lock file.

Reply:
`uv run` installs the correct environment including python.

Reply:
A single location for pinning the dependencies has my preference so I removed the inline script metadata. Also I adopted the --frozen flag to avoid updates to the environment.